src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -217,17 +213,15 @@ public static void itemRender(RenderHandItemEvent.Pre event) {
                 event.setCanceled(true);
             }
 
-            Item item1 = player.getMainHandItem().getItem();
-            Item item2 = player.getOffhandItem().getItem();
-
-            if (item1 instanceof VehicleItem) {
-                if (event.getHandSide() == HumanoidArm.LEFT) {
+            if (event.getHandSide() == HumanoidArm.LEFT) {
+                Item mainHandItem = player.getMainHandItem().getItem();
+                if (mainHandItem instanceof VehicleItem) {
                     event.setCanceled(true);
                 }
-            }
-
-            if (item2 instanceof VehicleItem) {
-                if (event.getHandSide() == HumanoidArm.RIGHT) {
+            } else {
+                // HumanoidArm.RIGHT
```

Comment:
`event.getHandSide()` will only return *one of* left or right (can't be both simultaneously in the same event fire), so we can safely assume it's `HumanoidArm.RIGHT` if it's not left.